### ISSUE TITLE
After TB update to 96.0b1 (64-bit) - DKIM verifier header line dissapeared.. Error

### ISSUE THREAD

**Issue by xsistema** [lieser/dkim_verifier#279]
After TB update to 96.0b1 (64-bit) - DKIM verifier header line dissapeared.. Error:

```
Uncaught Error: Could not find the DKIMHeaderField element
    get jar:file:///C:/Users/xxx/AppData/Roaming/Thunderbird/Profiles/default/extensions/dkim_verifier@pl.xpi!/experiments/dkimHeader.js:340
    reset jar:file:///C:/Users/xxx/AppData/Roaming/Thunderbird/Profiles/default/extensions/dkim_verifier@pl.xpi!/experiments/dkimHeader.js:764
    onStartHeaders jar:file:///C:/Users/xxx/AppData/Roaming/Thunderbird/Profiles/default/extensions/dkim_verifier@pl.xpi!/experiments/dkimHeader.js:773
    onStartHeaders chrome://messenger/content/msgHdrView.js:589
    processHeaders chrome://messenger/content/msgHdrView.js:641
dkimHeader.js:340:10
This page is in Quirks Mode. Page layout may be impacted. For Standards Mode use “<!DOCTYPE html>”. Inbox
Could not find the DkimHeaderRow element dkimHeader.js:414
DKIM_Verifier.AuthVerifier	DEBUG	 AuthResult result found:  {"version":"3.0","dkim":[{"version":"2.0","result":"SUCCESS","sdid":"ebay.com","auid":"@ebay.com","selector":"dkim1k","warnings":[{"name":"DKIM_SIGERROR_INSECURE_A"}],"keySecure":false}]} authVerifier.mjs.js:325:6
Error: An unexpected error occurred undefined
Could not find the DKIMHeaderField element dkimHeader.js:340
DKIM_Verifier.background	FATAL	 Unexpected error during verifyMessage <unavailable> background.mjs.js:133:7
Could not find the DkimHeaderRow element dkimHeader.js:414
Could not find the DKIMHeaderField element dkimHeader.js:340
Error: An unexpected error occurred 2 undefined

```

**Comment by DerFisc** [comment]
Same for me. 
It even seems to make the complete header field with from field, reply button, ... disappear for the message pane.

**Comment by lieser** [comment]
Thanks you both for reporting the issue. Had a quick look, and unfortunately this is more than a one line fix.
Seems like Thunderbird is in the progress of changing some elements in the email header, which also requires some changes to the elements the addon is adding.

Will try to fix it in the next few days.

**Comment by lieser** [comment]
I was able to fix the DKIM header line, and also the favicon which had problems too.

The fix will be included in 4.1.0. Unfortunately I don't think that version will still get released this year.
But I attached a build of the current master that you can already use.
[dkim_verifier@pl-b2d699a.zip](https://github.com/lieser/dkim_verifier/files/7740804/dkim_verifier%40pl-b2d699a.zip)
